packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -126,10 +128,14 @@ class _ToolbarContainerLayout extends SingleChildLayoutDelegate {
 class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// Creates a material design app bar.
   ///
+  /// Arguments [elevation], [primary], [toolbarOpacity], [bottomOpacity],
+  /// [automaticallyImplyLeading], must not be null.
```

Comment:
Thanks for adding this. Can you add a "The" at the stand and an "and" between the last two and remove the final comma, to match the grammar used elsewhere? Thanks! (If this is the style you found somewhere, please let me know and I'll go fix it there too.)